.github/workflows/upstream-issues.yml

```diff
@@ -0,0 +1,15 @@
+name: Upstream issue notifier
+on:
+  schedule:
+  # run at 8am EST every day
+    - cron: "0 13 * * *"
+jobs:
+  upstream-issues:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Kyle-Verhoog/upstream-issue-notifier@v0.1.1
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+          IGNORE_DIRS: ddtrace/vendor
+          DRY_RUN: false
```

Comment:
```suggestion
```

I think this actually keeps DRY_RUN enabled since it just checks if the variable is defined or not.